core/spec.md

```diff
@@ -86,6 +86,11 @@ to a "Schema" document - all within the same Registry instance:
  src="./xregfullmodel.png" height="300">&nbsp;&nbsp;&nbsp;<img
  src="./sample.png" height="300">
 
+One purpose of the xRegistry is to enable easy and fully automated,
+machine-readable discovery. The starting point of the discovery is via a
+[well-known location](#registry-well-known-config): `/.well-known/xregistry.json`.
+This is relative to the base URL of the server.
```

Comment:
"of the server". Does this mean of the host or of the xReg server? Meaning: http://example.com or http://example.com/xregistry  if the latter is the base URL of the xReg server?

Reply:
@duglin : In my understanding example.com would be the location to the server (and this would be its base url / hostname). /xregistry would be a path that actually is the entry point of an API that the server offers (but there may be other APIs as well).

Reply:
Based on today's discussion I assume line 92 would change to allow for it to live some place else, right?

Reply:
> Based on today's discussion I assume line 92 would change to allow for it to live some place else, right?

I don't think so: The .well-known URI needs to be first discovered. But when you retrieve it, then you can point to the actual xRegistry server implementation(s) and their URLs. They could indeed live on a different server.